packages/app-mobile/components/NoteList.tsx

```diff
@@ -90,7 +90,11 @@ class NoteListComponent extends Component<NoteListProps> {
 				keyExtractor={item => item.id}
 			/>;
 		} else {
-			if (!this.props.folders.length) {
+			// calculate number of folders except deleted and trash folder
+			// trash folder id: de1e7ede1e7ede1e7ede1e7ede1e7ede
+			const atleastOneNotebookExists = this.props.folders.filter((folder: FolderEntity) => folder.id !== 'de1e7ede1e7ede1e7ede1e7ede1e7ede' && folder.deleted_time === 0).length > 0;
```

Comment:
Consider using `getTrashFolderId` here instead of `de1e7ede1e7ede1e7ede1e7ede1e7ede`. ([Defined here](https://github.com/laurent22/joplin/blob/52810c51f53106f42eef891ca2500abb941c7893/packages/lib/services/trash/index.ts#L36)).

Thank you for working on this!

Reply:
Hey, I have made the suggested changes and updated the pr. Do let me know if any other changes are needed!